project/Build.scala

```diff
@@ -12,7 +12,7 @@ object DottyBuild extends Build {
   val baseVersion = "0.1"
   val isNightly = sys.env.get("NIGHTLYBUILD") == Some("yes")
 
-  val jenkinsMemLimit = List("-Xmx1300m")
+  val jenkinsMemLimit = List("-Xmx1500m")
```

Comment:
Would you mind explaining the bug?
https://github.com/lampepfl/dotty/pull/1636/commits/7f4f387f220478ce4f8f13b99c7d16f0c254339d 
does a lot of changes but I don't see anything particularly related to `-deep` or something similar.

The biggest tests that we had so far were stdlib and dotty. Stdlib relies on whitelist, and `dotty` did use `-deep` flag. If this flag works unreliably, it would be nice to know as other forks depend on it.